packages/url-loader/package.json

```diff
@@ -19,29 +29,17 @@
     "test": "vitest run",
     "test:watch": "vitest"
   },
-  "devDependencies": {
-    "@babel/core": "^7.20.12",
-    "@babel/preset-env": "^7.20.2",
-    "@cloudinary-util/tsconfig": "workspace:*",
-    "@colbyfayock/semantic-release-pnpm": "^1.2.2",
-    "@semantic-release/changelog": "^6.0.2",
-    "@semantic-release/git": "^10.0.1",
-    "@types/node": "^17.0.12",
-    "eslint": "^7.32.0",
-    "eslint-config-cloudinary-util": "workspace:*",
-    "semantic-release": "^20.1.0",
-    "semantic-release-monorepo": "^7.0.5",
-    "tsup": "^8.0.1",
-    "typescript": "^5.3.3",
-    "vitest": "^1.2.0"
-  },
-  "publishConfig": {
-    "access": "public"
-  },
   "dependencies": {
     "@cloudinary-util/types": "workspace:*",
     "@cloudinary-util/util": "workspace:*",
     "@cloudinary/url-gen": "1.15.0",
-    "zod": "^3.22.4"
+    "zod": "^3.22.4",
+    "zod-to-ts": "^1.2.0"
```

Comment:
this should be a dev dep

Reply:
Whoops, my bad! Just pushed a commit fixing this.